spring-grpc-docs/src/main/antora/modules/ROOT/partials/_configprops.adoc

```diff
@@ -0,0 +1,18 @@
+|===
```

Comment:
This file is automatically generated. Should we even check it in? I notice other spring-cloud repos are checking it in. I believe having it checked in avoids annoying red warnings coming from the consuming adocs that are referencing it.